### PR TITLE
chore: Clean up anchoring initialization

### PR DESCRIPTION
**Pull request by ukstv**
- No anchorService in context,
- validator is a part of AnchorService,
- AnchorService creation through opaque functions.

### CODE REVIEW

**Review by stephhuynh18 — APPROVED** [review]
🎈

File: packages/core/src/ceramic.ts [hunk 1]
```diff
@@ -240,12 +226,11 @@ export class Ceramic implements CeramicApi {
 
     this.context = {
       api: this,
-      anchorService: modules.anchorService,
       ipfs: modules.ipfs,
       loggerProvider: modules.loggerProvider,
     }
     if (!this._gateway) {
-      this.context.anchorService.ceramic = this
+      this.anchorService.ceramic = this
```
Comment (stbrody) [inline packages/core/src/ceramic.ts:233]: 😢

**Review by stbrody — APPROVED** [review]
nice

File: packages/core/src/ceramic.ts (hunk 1 above)
Comment (ukstv) [reply packages/core/src/ceramic.ts:233]: A necessary evil now, that we'll change someday. In prod it is a no-op though, so not as harmful as it looks.

**Pull request merged**